packages/env-loader/src/index.js

```diff
@@ -16,16 +16,12 @@ const config = path => {
     files.push(path);
   }
 };
-config("env/.env.local");
-config("env/.env");
-
-const deploymentEnv = process.env.DEPLOYMENT_ENV;
 
-if (deploymentEnv && deploymentEnv !== "local") {
-  config(`env/.env.${deploymentEnv}`);
+if (!process.env.DEPLOYMENT_ENV) {
```

Comment:
can also be `local`

Reply:
only if set on system level. the first file which sets this variable is project level `.env.local` currently. So, if there is no this variable at system level (what is very likely for local env), we load extra file which can set `DEPLOYMENT_ENV` to whatever value we want. If it doesn't set, then project specific will set value for this env var